config/paths.js

```diff
@@ -12,11 +12,8 @@ module.exports = {
   root: appDirectory,
   dotenv: resolveApp(".env"),
   favicon: resolveApp("src/assets/logo/rasterized/logo_32.png"),
-  appBuild: resolveApp("build"),
   appBuild2: resolveApp("build2"),
```

Comment:
Nit, should we now rename appBuild2 to appBuild ?

Reply:
I'll do it in a followon.

Reply:
cf https://github.com/sourcecred/sourcecred/pull/1882